types/matomo-tracker-react-native/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for matomo-tracker-react-native 0.3
+// Project: https://github.com/donni106/matomo-tracker-react-native
+// Definitions by: Zeynep Ece Ergin <https://github.com/zecergin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
+
+import * as React from 'react';
+
+export interface AppStart {
+  userInfo?: {
+    uid?: string
+  };
+}
+
+export interface Action {
+  name: string;
+  userInfo?:
+  {
+    uid?: string
+  };
```

Comment:
```suggestion
  userInfo?:
  {
    uid?: string
  };
```
this block is recurring theme